packages/daemon/src/migrations/20200803063219-addDatasetProcess.ts

```diff
@@ -6,11 +6,12 @@ export default {
     return queryInterface.sequelize.transaction(async t => {
       const tableNames = await queryInterface.showAllTables();
       if (tableNames.indexOf('pipelines') >= 0) {
+        const tables = await queryInterface.describeTable('pipelines');
```

Comment:
It can be renamed to `table`?

Reply:
It will get all tables under 'pipelines', `tables` might be the most suitable name